### DISCUSSION TITLE
Cache Tag Does Not Render Strings

### DISCUSSION THREAD

**Discussion by martbock** [statamic/cms#8394] [Troubleshooting]
I have an Antlers template that uses a user-scoped cache tag to render HTML content wrapped in partials. For some reason, Antlers renders obscure IDs like `__literalReplacement::_3bc4ba0c73ddc55b90df9094c7d2676e` (cache identifier?) instead of the intended strings.

Does anyone have a tip on what could be wrong here?

Shortened template example:

```antlers
{{ cache scope="user" }}
    ...
    {{ partial:card }}
        {{ section:header }}
            {{ user }}
                <div>
                    Hello {{ name }}
                </div>
            {{ /user }}
        {{ /section:header }}
    {{ /partial:card }}
    ...
{{ /cache }}
```

First render (uncached): `Hello Jon Doe`
Subsequent renders (cached): `__literalReplacement::_3bc4ba0c73ddc55b90df9094c7d2676e`

**Comment by jackmcdade** [comment]
Is that {{ cache }} tag in a layout, a template, or a partial? If you move it elsewhere in your template stack does it work?

**Comment by martbock** [comment]
It's in a template. After testing some more, it appears the `cache` tag somehow interferes with the `section` tag. 
It works if I move the `cache` tag into the layout. It also works if I move it inside the `section` tag. But as soon as I try to use `cache` in a template where  it contains a `partial` with a `section` tag, the inner text gets swapped.

Is this intended behavior and I'm abusing something here? Or maybe a bug? Not quite sure... 😄 

I guess a workaround could be to put `cache` tags into every `section` of the layout.
